powered-by.md

```diff
@@ -117,6 +117,9 @@ and external data sources, driving holistic and actionable insights.
   - We are using Spark for analyzing and visualizing patterns in large-scale recordings of brain 
   activity in real time
 - <a href="http://www.fundacionctic.org">Fundacion CTIC</a>
+- <a href="https://graal.systems">GraalSystems</a>
+  - GraalSystems is a cloud-native data platform that can be used erverywhere, on cloud 
```

Comment:
`erverywhere` -> `everywhere`?

Reply:
Oops, missed that - if the OP wants to make another PR I'll merge

Reply:
Oops, sorry for the typo...